README.md

```diff
@@ -252,15 +252,15 @@
        1. [ ] What is REST API? 
        2. [ ] What is GraphQL API? 
    1. [ ] CRUD operations
-   1. [ ] What are cookies? 
-      1. [ ] Getting and setting cookies
-   1. [ ] What is local storage?
-   1. [ ] What is session storage?
-   1. [ ] How does a web browser work? 
-   1. [ ] What is server vs client?
-   1. [ ] What is a domain name?
-   1. [ ] What is DNS lookup?
-   1. [ ] What is web hosting?
+   1. [ ] What are cookies? [Link](https://www.cloudflare.com/en-gb/learning/privacy/what-are-cookies/)
+      1. [ ] Getting and setting cookies [Link](https://flaviocopes.com/cookies/)
+   1. [ ] What is local storage? [Link](https://blog.logrocket.com/localstorage-javascript-complete-guide/)
+   1. [ ] What is session storage? [Link](https://www.javascripttutorial.net/web-apis/javascript-sessionstorage/)
+   1. [ ] How does a web browser work? [Link](https://www.youtube.com/watch?v=uE3UPEK26U0&ab_channel=MozillaHacks)
```

Comment:
One small change: Could you change [Link] to [Video] for the youtube links? Will be easier for readers to know which ones are videos. Thank you.